scripts/Muon/GUI/Common/home_instrument_widget/home_instrument_widget_model.py

```diff
@@ -63,13 +63,19 @@ def get_user_first_good_data(self):
             first_good_data = self._context.gui_context["FirstGoodData"]
         return first_good_data
 
-    def get_last_good_data(self):
+    def get_file_last_good_data(self):
         if self._data.current_runs:
             run = self._data.current_runs[0]
             return self._context.last_good_data(run)
         else:
             return 0.0
 
+    def get_last_good_data(self):
```

Comment:
why do we need this?

Reply:
This is to get last good if the user is supplying the value, instead of from file. They are separate to make the GUI easier to reset and update as you'd expect

Reply:
It also keeps them as two separate inputs which can be distinguished by an if statement, as the new changes show